frontend/app/src/SegmentMetricsManager.ts

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+import {
+  isNullOrUndefined,
+  notNullOrUndefined,
+} from "@streamlit/lib/src/util/utils"
 import pick from "lodash/pick"
 import {
   Delta,
```

Comment:
I think in a followup perhaps we should ensure that @\streamlit gets treated as an internal import? Idk.. thoughts?

Reply:
100%!